core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -20,6 +20,8 @@
 #include <type_traits>
 #include <initializer_list>
 
+#include <Kokkos_Array.hpp>            // is_zero_byte
```

Comment:
```suggestion
#include <Kokkos_Array.hpp>
```

Reply:
Do you plan on getting rid of these comments?  See https://github.com/kokkos/kokkos/pull/7014#pullrequestreview-2066437540

Reply:
Done.